app/javascript/src/components/Dashboard/Contacts/NewContactForm.jsx

```diff
@@ -0,0 +1,112 @@
+import { Input, Textarea } from "neetoui/v2/formik";
+import * as yup from "yup";
+
+import notesApi from "apis/notes";
+
+export default function NewContactForm({ onClose, refetch }) {
+  const handleSubmit = async values => {
+    try {
+      await notesApi.create(values);
+      refetch();
+      onClose();
+    } catch (err) {
+      logger.error(err);
+    }
+  };
+  return (
+    <Formik
+      initialValues={{
+        title: "",
+        description: ""
+      }}
+      onSubmit={handleSubmit}
+      validationSchema={yup.object({
+        title: yup.string().required("Title is required"),
+        description: yup.string().required("Description is required")
+      })}
+    >
+      {() => (
+        <Form>
+          <Input
+            label="Title"
+            name="title"
+            className="mb-6"
+            required={true}
+            placeholder="Enter Note Title"
+          />
+          <Textarea
+            label="Description"
+            name="description"
+            rows={1}
+            required={true}
+            placeholder="Enter Note Description"
+          />
+          <div className="my-6">
+            <Select
+              isClearable
+              isSearchable
+              required={true}
+              label="Assigned Contact"
+              name="role"
+              options={[
+                {
+                  label: "Value One",
+                  value: "value1"
+                },
+                {
+                  label: "Value Two",
+                  value: "value2"
+                },
+                {
+                  label: "Value Three",
+                  value: "value3"
+                },
+                {
+                  label: "Value Four",
+                  value: "value4"
+                },
+                {
+                  label: "Value Five",
+                  value: "value5"
+                }
+              ]}
+              placeholder="Select a Role"
+            />
+          </div>
+          <div className="my-6">
+            <Select
+              isClearable
+              isSearchable
+              required={true}
+              isMulti
+              label="Tags"
+              name="tags"
+              options={[
+                {
+                  label: (
+                    <Tag color="green" label="Getting Started" size="small" />
+                  ),
+                  value: "Getting Started"
+                },
+                {
+                  label: <Tag color="blue" label="Tag Two" size="small" />,
+                  value: "Tag Two"
+                },
+                {
+                  label: <Tag color="red" label="Tag Three" size="small" />,
+                  value: "Tag Three"
+                }
+              ]}
+              placeholder="Select Role"
+            />
+          </div>
+          <div className="nui-pane__footer nui-pane__footer--absolute"></div>
+        </Form>
```

Comment:
Please use `space-y-*` utils from tailwindCSS instead of `mb-6` for individual blocks.